docs/content/misc/contribute.ngdoc

```diff
@@ -24,20 +24,27 @@ Before you can build AngularJS, you must install and configure the following dep
 machine:
 
 * [Git](http://git-scm.com/): The [Github Guide to
-Installing Git](https://help.github.com/articles/set-up-git) is a good source of information.
+  Installing Git](https://help.github.com/articles/set-up-git) is a good source of information.
 
-* [Node.js v4.x](http://nodejs.org): We use Node to generate the documentation, run a
-development web server, run tests, and generate distributable files. Depending on your system, you can install Node either from source or as a
-pre-packaged bundle. (Currently our build does not work properly on Node v5 or greater - please use v4.x.)
+* [Node.js v6.x (LTS)](http://nodejs.org): We use Node to generate the documentation, run a
+  development web server, run tests, and generate distributable files. Depending on your system,
+  you can install Node either from source or as a pre-packaged bundle.
+
+  We recommend using [nvm](https://github.com/creationix/nvm) to manage and install Node.js,
```

Comment:
I've been using https://github.com/coreybutler/nvm-windows for some time now and it works fine. I think it is worth mentioning it too.